### PR TITLE
Changed first commit

### PR DESCRIPTION
**Pull request by fortunat-a-ly**
add task lists, execution, creation, timers

### CODE REVIEW

**Comment by DanielJorn** [comment]
add .idea to .gitignore

File: app/src/main/java/com/example/activitytimer/MainActivity.kt [hunk 1]
```diff
@@ -26,9 +32,15 @@ class MainActivity : AppCompatActivity() {
 
         setSupportActionBar(binding.toolbar)
 
+        Log.d("ViewP", "main")
+        val adapter = ViewPagerAdapter(supportFragmentManager)
+        binding.pager.adapter = adapter
+        binding.tabLayout.setupWithViewPager(binding.pager)
+        Log.d("ViewP", "after ViewP")
+/*
```
Comment (DanielJorn) [inline app/src/main/java/com/example/activitytimer/MainActivity.kt:40]: dont leave commented code - delete it. you can get them from git history if you need them later (but you probably won't)

File: app/src/main/java/com/example/activitytimer/SearchableActivity.kt [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import androidx.appcompat.app.AppCompatActivity
+import androidx.appcompat.widget.SearchView
+import androidx.core.content.ContextCompat
+
+class SearchableActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        setContentView(R.layout.dialog_choose_category)
+        //this.setFinishOnTouchOutside(false)
```
Comment (DanielJorn) [inline app/src/main/java/com/example/activitytimer/SearchableActivity.kt:16]: the same

File: app/src/main/java/com/example/activitytimer/data/doneTasks/DoneTaskDatabaseDao.kt [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+import androidx.room.Dao
+import androidx.room.Insert
+import androidx.room.Query
+
+@Dao
+interface DoneTaskDatabaseDao {
+    @Insert
+    fun insert(task: DoneTask) : Long
+
+/*    @Query("SELECT * FROM done_task_table ORDER BY date DESC")
```
Comment (DanielJorn) [inline app/src/main/java/com/example/activitytimer/data/doneTasks/DoneTaskDatabaseDao.kt:13]: see above

File: app/src/main/java/com/example/activitytimer/data/subtask/SubtaskDatabaseDao.kt [hunk 4]
```diff
@@ -4,18 +4,31 @@ import androidx.lifecycle.LiveData
...
+    @Insert
+    fun insert(subtask: List<Subtask>)
+
+    @Query("SELECT * from subtask_table")
+    fun getAll(): List<Subtask>?
+
     @Query("SELECT * from subtask_table WHERE id = :key")
     fun get(key: Long): Subtask?
+/*
+    @Query("SELECT * FROM subtask_table WHERE task_id = :taskKey ORDER BY id ASC")
```
Comment (DanielJorn) [inline app/src/main/java/com/example/activitytimer/data/subtask/SubtaskDatabaseDao.kt:24]: c up

File: app/src/main/java/com/example/activitytimer/screens/createTask/ChooseCategoryDialogFragment.kt [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+    private var checkedId: Int = -1
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        val view = inflater.inflate(R.layout.dialog_choose_category, container)
+
+/*        val searchManager = getSystemService(requireContext(), SearchManager::class.java)
```
Comment (DanielJorn) [inline app/src/main/java/com/example/activitytimer/screens/createTask/ChooseCategoryDialogFragment.kt:31]: 2 many comments

File: app/src/main/java/com/example/activitytimer/screens/createTask/CreateSubtaskFragment.kt [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+                navController.popBackStack()
+                Log.d("BugH", "after click")
+            }
+        }
+
+        return binding.root
+    }
+
+    private fun readInput() {
+        // viewModel.subtask.time = binding.editTextTime.text.toString().toLong()
```
Comment (DanielJorn) [inline app/src/main/java/com/example/activitytimer/screens/createTask/CreateSubtaskFragment.kt:56]: delete

**Comment by fortunat-a-ly** [comment]
> add .idea to .gitignore

https://intellij-support.jetbrains.com/hc/en-us/articles/206544839-How-to-manage-projects-under-Version-Control-Systems

**Comment by fortunat-a-ly** [comment]
> add .idea to .gitignore

https://intellij-support.jetbrains.com/hc/en-us/articles/206544839-How-to-manage-projects-under-Version-Control-Systems

**Pull request closed without merging**